### DISCUSSION TITLE
Anyone have recommendations for nice frameworks to build frontend components for NocoDB?

### DISCUSSION THREAD

**Discussion by danielrosehill** [nocodb/nocodb#9308] [General]
Firstly, congrats to the Noco team for a great update (at least I just saw it!). Looks like some cool things are in the works.

My database is coming along very nicely and I've got a basic handle on using the API.

All I'm wanting for now is ... something a little more "friendly" for interfacing with the data than NocoDB itself (by which I mean basically a tool for "end users" even though in this case the end user and me are the same overworked person!)

My database is essentially a huge clearing house of ChatGPT outputs. Mostly speccing out technical projects but a good amount of work stuff in there too:

![image](https://github.com/user-attachments/assets/b5b63cf1-0d96-423f-8bf1-197382430b38)

The UI elements I would like are:

- Make it easy to read the outputs on my phone (ie, a responsive web element or something similar)
- Make it easy to edit the outputs when I'm at the computer (ie, a backend / web UI)

Anyone have a recommended pairing (ideally also within the world of no code but that's not a hard requirement)

**Comment by lukecleland** [comment]
Hi @danielrosehill was wondering if you had any traction on this concept, was thinking of making some components of this kind myself. The linking capability would be particularly good to import/emulate.
